logistics_project/apps/api/resources/v0_1.py

```diff
@@ -68,12 +68,30 @@ class SupplyPointResources(ModelResource):
     active = fields.BooleanField('active')
     type = fields.CharField('type')
     location_id = fields.IntegerField('location_id', null=True)
+    products = fields.ToManyField(
+        ProductResources,
+        attribute=lambda
+            bundle: Product.objects.filter(
+                id__in=ProductStock.objects.filter(supply_point=bundle.obj,
+                                                   is_active=True,
+                                                   product__is_active=True).values_list(*['product'], flat=True),
```

Comment:
also, since you're just pulling out the `sms_codes` below can you do something like `.values_list(*['product__sms_code'], flat=True)`? (i'm not sure if that works, just wondering if we can optimize a bit since this will get called a fair amount.